x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/insight/index.tsx

```diff
@@ -238,28 +158,48 @@ const InsightComponent = ({
       }),
     });
   }
-  const { data: alertData } = useContext(BasicAlertDataContext);
+  const { data: alertData, timestamp } = useContext(BasicAlertDataContext);
   const { dataProviders, filters } = useInsightDataProviders({
     providers: parsedProviders,
     alertData,
   });
-  const { totalCount, isQueryLoading, oldestTimestamp, hasError } = useInsightQuery({
-    dataProviders,
-    filters,
-  });
-  const timerange: TimeRange = useMemo(() => {
+  const relativeTimerange: TimeRange | null = useMemo(() => {
     if (relativeFrom && relativeTo) {
       const fromStr = relativeFrom;
       const toStr = relativeTo;
-      const from = parseDateWithDefault(fromStr, DEFAULT_FROM_MOMENT).toISOString();
-      const to = parseDateWithDefault(toStr, DEFAULT_TO_MOMENT, true).toISOString();
+      const alertRelativeDate = timestamp ? moment(timestamp) : moment();
```

Comment:
also not sure what's the point of setting the 2 variables above (fromStr and toStr)

Reply:
those are the names the action payload that makes use of the timeline expects, and made more sense when those were returned, but will remove